xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1177,9 +1233,9 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The group to locate in the group collection.</param>
+        <summary>Determines whether the group collection contains a specific group item.</summary>
+        <returns><see langword="true" /> if the group item is found in the group collection; <see langword="false" /> otherwise.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. 

         ]]></format>
         </remarks>
```

Reply:
Edited your suggestion to add the `## Remarks` title.